controller/api/public/stat_summary.go

```diff
@@ -0,0 +1,327 @@
+package public
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/prometheus/common/model"
+	apiUtil "github.com/runconduit/conduit/controller/api/util"
+	pb "github.com/runconduit/conduit/controller/gen/public"
+	"github.com/runconduit/conduit/pkg/k8s"
+	log "github.com/sirupsen/logrus"
+	appsv1 "k8s.io/api/apps/v1"
+	apiv1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/fields"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+const (
+	reqQuery = "sum(increase(response_total{%s}[%s])) by (%s, classification)"
+)
+
+type meshedCount struct {
+	inMesh    uint64
+	notInMesh uint64
+}
```

Comment:
Totally minor, but since we want to return MeshedPodCount and TotalPodCount in the protobuf, it might make sense to track those two numbers directly, rather than needing to derive total by adding these two fields together. We're not using the notInMesh number in any other context.

I'd change this to:

```
type meshedCount struct {
	inMesh    uint64
	total uint64
}
```

And then inside the getMeshedPodCount func, you can do:

```
	for _, pod := range pods {
		meshCount.total++
		if isInMesh(pod) {
			meshCount.inMesh++
		}
	}
```